common/app/Nav/components/NavLinks.jsx

```diff
@@ -40,7 +40,7 @@ function mapDispatchToProps(dispatch) {
           const handler = `handle${capitalize(content)}Click`;
           mdtp[handler] = createNavLinkActionCreator(content);
           return mdtp;
-        }),
+        }, []),
```

Comment:
`mapDispatchToProps` is expected to return an object.

With this change we will be destructuring an array into an object and it would look like
```js
{
  0: [handler],
  1: [handler]
  // ...
}
```

This reduce function should be initialized with an object.

Reply:
Ah yes, the bracket notation for object properties had me fooled into thinking it was an array. I'm assuming I can initialized with an empty object `{}`? It means if the links array is empty, then an empty object will be returned by the `reduce`. If you can confirm I'll get this fixed up. Thanks for the review!

Reply:
Yes, you can initalise it with an empty object.
One question, why would we have an empty links array? I'm a bit sketchy in this area of the code, it has been a while

Reply:
Fixed, and I think this is ready for merge now.

To answer your question, the links array will probably never be empty in practice, but if for some reason you needed to temporarily remove the links in production, the app would crash. With this fix it will just work in all cases.